src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -789,30 +789,212 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
+                                            if ($myinv.ScriptName) {
+                                                $posmsg = ""error in${resetcolor} $($myinv.ScriptName)${newline}""
+                                            }
+                                            else {
+                                                $posmsg = ""${newline}""
+                                            }
+
+                                            $scriptLineNumberLength = $myinv.ScriptLineNumber.ToString().Length
+                                            if ($scriptLineNumberLength -gt 4) {
+                                                $headerWhitespace = ' ' * ($scriptLineNumberLength - 4)
+                                            }
+
+                                            $lineWhitespace = ''
+                                            if ($scriptLineNumberLength -lt 4) {
+                                                $lineWhitespace = ' ' * (4 - $scriptLineNumberLength)
+                                            }
+
+                                            $verticalBar = '|'
+                                            $posmsg += ""${accentColor}${headerWhitespace}Line ${verticalBar}${newline}""
+                                            $line = $myinv.Line
+                                            $highlightLine = $myinv.PositionMessage.Split('+').Count - 1
+                                            $offsetLength = $myinv.PositionMessage.split('+')[$highlightLine].Trim().Length
+                                            $line = $line.Insert($myinv.OffsetInLine - 1 + $offsetLength, $resetColor)
+                                            $line = $line.Insert($myinv.OffsetInLine - 1, $errorColor)
+                                            $posmsg += ""${accentColor}${lineWhitespace}$($myinv.ScriptLineNumber) ${verticalBar} ${resetcolor}${line}`n""
+                                            $offsetWhitespace = ' ' * ($myinv.OffsetInLine - 1)
+                                            $prefix = ""${accentColor}${headerWhitespace}     ${verticalBar} ${errorColor}""
+                                            $message = ""${prefix}${offsetWhitespace}^ ""
+                                        }
+
+                                        if (! $_.ErrorDetails -or ! $_.ErrorDetails.Message) {
+                                            # we use `n instead of $newline here because that's what is in the message
+                                            if ($_.CategoryInfo.Category -eq 'ParserError' -and $_.Exception.Message.Contains(""~`n"")) {
+                                                # need to parse out the relevant part of the pre-rendered positionmessage
+                                                $message += $_.Exception.Message.split(""~`n"")[1].split(""${newline}${newline}"")[0]
```

Comment:
`""~`n""` not `${newline}`?

Reply:
I added a comment on 910, since it seems that regardless if Windows or macOS, from the resx file it is a `` `n``